src/Test/Perf/util/test_util.csx

```diff
@@ -17,33 +17,36 @@ using System.Net;
 // Arguments
 //
 
+// This is due to a "bug" in csi that has Args non-static.  Thus, you can't 
+// use it from non-static methods.
```

Comment:
Is there a bug on this?  In general I find it kind of annoying that I can't reference TLDs from within a class declaration (forcing us to add ```static```, which forces us to do this).